waku/v2/protocol/waku_store/protocol.nim

```diff
@@ -133,35 +130,34 @@ proc findMessages(w: WakuStore, query: HistoryQuery): HistoryResponse {.gcsafe.}
 proc init*(ws: WakuStore, capacity = StoreDefaultCapacity) =
 
   proc handler(conn: Connection, proto: string) {.async.} =
-    var message = await conn.readLp(MaxRpcSize.int)
-    var res = HistoryRPC.init(message)
-    if res.isErr:
-      error "failed to decode rpc"
+    let buf = await conn.readLp(MaxRpcSize.int)
+
+    let resReq = HistoryRPC.init(buf)
+    if resReq.isErr():
+      error "failed to decode rpc", peerId=conn.peerId
       waku_store_errors.inc(labelValues = [decodeRpcFailure])
       return
 
-    # TODO Print more info here
-    info "received query", rpc=res.value
+    let req = resReq.value
+
+    info "received history query", peerId=conn.peerId, requestId=req.requestId, query=req.query
```

Comment:
Note: nice touch with `requestId` 👍 . I understand it should allow grouping of the logs when looking up.